libs/feature/search/src/lib/fuzzy-search/fuzzy-search.component.spec.ts

```diff
@@ -212,4 +212,23 @@ describe('FuzzySearchComponent', () => {
       })
     })
   })
+
+  describe('custom placeholder', () => {
+    beforeEach(() => {
+      fixture = TestBed.createComponent(FuzzySearchComponent)
+      component = fixture.componentInstance
+      component.placeholder = 'Type your custom placeholder text here…'
```

Comment:
Maybe we could add another test without setting the component's input and verify if the `autocomplete.component` receives well `search.field.any.placeholder`? I let you decide.